src/main/java/com/blackducksoftware/ohcount4j/FileCount.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2016 Black Duck Software, Inc.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.blackducksoftware.ohcount4j;
+
+public class FileCount {
+    private final SourceFile source;
+    private final LanguageCount count;
```

Comment:
@PeterDP I actually mimicked the existing API. The domain terminology was not very clear to me, so I went with what I understood first. I'll try renaming the details of this VO so that it makes more sense.